inspirehep/modules/authors/dojson/fields/updateform.py

```diff
@@ -186,7 +186,7 @@ def institution_history(self, key, value):
             continue
         positions.append({
             "institution": {'name': position["name"]},
-            "current": 'Current' if position["current"] else None,
+            "current": 'Current' if position["current"] else False,
```

Comment:
You are right, in the schema is defined as `boolean` so should be failing when passing a string.